packages/compass-settings/src/components/settings/theme.tsx

```diff
@@ -29,13 +29,27 @@ const checkboxStyles = css({
   marginBottom: spacing[3],
 });
 
+const radioBoxStyles = css({
+  div: {
```

Comment:
Bit of a hack, but I think so is sticking an image plus text in here :)

Reply:
We could have more control by only having an image as the content of the radio button. Maybe a task for a follow-up.